website/addons/twofactor/views.py

```diff
@@ -25,3 +26,37 @@ def user_settings(user_addon, *args, **kwargs):
         message_short='Forbidden',
         message_long='The two-factor verification code you provided is invalid.'
     ))
+
+@must_be_logged_in
+def twofactor_settings_get(auth, *args, **kwargs):
+    return {
+        'result': serialize_settings(auth),
+    }
+
+
+@must_be_logged_in
+def twofactor_enable(auth, *args, **kwargs):
+    user = auth.user
+
+    if user.has_addon('twofactor'):
+        return HTTPError(http.BAD_REQUEST, message='This user already has twofactor enabled')
```

Comment:
Also, keep the casing and hyphenation consistent in messages passed to the client; I believe I saw "Two-factor" in most other places, but make sure they're consistent.